Python/Product/PythonTools/PythonTools.csproj

```diff
@@ -198,6 +198,9 @@
       <Private>True</Private>
       <IncludeInVSIX>True</IncludeInVSIX>
     </Reference>
+    <Reference Include="StreamJsonRpc">
```

Comment:
please search for StreamJsonRpc in `Build\Common.Build.CSharp.targets` 
you should be able to change `net46` there to `netstandard2.0` and undo this change, as well as undo your change in PreBuild.ps1

Reply:
ah, thank you. I guess I missed this one when I searched for StreamJsonRpc string in the folder.